src/utils/printCreditsCppBids.m

```diff
@@ -2,23 +2,15 @@
 
 function printCreditsCppBids(cfg)
     %
-    % Short description of what the function does goes here.
+    % It will print the credits of this repo. Depending on the level of verbosit set in
+    % ``cfg.verbose`` (default is 2 if not set), it will print the graphic and general information.
```

Comment:
I was curious, do you know where can I see different levels of cfg.verbose? maybe it's in another function?

Reply:
Do you mean what is printed or not for each level? Good question, I have no idea

Reply:
Well they are not specified anywhere yet. But I was checking and it seems :
- 0: almost nothing gets printed on screen 
- 1: allows `saveEvents` to print stuff when saving to file or `createFilename` to tell you where things will be saved
- 2: let `printCredits` do its job and lets `saveEvents` print extra info when it sends some warning.

Reply:
I think that belongs more in the doc in genereal than in the help section of any specific function. What do you think?